hie-ghc-mod/Haskell/Ide/GhcModPlugin.hs

```diff
@@ -130,9 +138,23 @@ typeCmd = CmdSync $ \_ctxs req ->
   case getParams (IdFile "file" :& IdPos "start_pos" :& RNil) req of
     Left err -> return err
     Right (ParamFile fileName :& ParamPos (Pos (Line l) (Col c)) :& RNil) -> do
-      fmap (toTypeInfo . T.lines . T.pack) <$> runGhcModCommand (GM.types (T.unpack fileName) l c)
+      fmap (toTypeInfo . T.lines . T.pack) <$> runGhcModCommand (GM.types False (T.unpack fileName) l c)
+#if __GLASGOW_HASKELL__ <= 710
     Right _ -> return $ IdeResponseError (IdeError InternalError
       "GhcModPlugin.typesCmd: ghc’s exhaustiveness checker is broken" Null)
+#endif
+
+{-
+TODO: pass the bool through as a param
```

Comment:
It might make more sense to move this TODO directly to the call of types to make it easier to understand.